### ISSUE TITLE
Use youtube data

### ISSUE THREAD

**Issue by olypros** [saurabhmathur96/clickbait-detector#7]
Can you scrap the youtube channels all video titles known for clickbaits example( leafy,reactions,progamer etc ) and non clickbait chanels (idk ask reddit) and finetune the model ?

**Comment by saurabhmathur96** [comment]
The model can definitely be fine tuned. You can use the Youtube data API to fetch titles of videos by particular channels (https://developers.google.com/youtube/v3/docs/videos/list).
Similarly, Reddit has an API too (https://www.reddit.com/dev/api/) from which you can get post titles

**Comment by olypros** [comment]
Yes I know that it can be fine tuned.  But I'm don't know any ML.  So it
would be great if you create an example script to fine tune model. You
don't have to use api just a small example associative array (or dict)
containing title and its respective label.

Thanks

**Comment by saurabhmathur96** [comment]
I had some hand-labelled data from reddit (/r/savedyouaclick and /r/news). So I made a notebook showing how to fine tune the model on that data. [Find the notebook here](http://nbviewer.jupyter.org/github/saurabhmathur96/clickbait-detector/blob/master/notebooks/Finetuning%20on%20Reddit%20data.ipynb). Hope this helps.
